usecases/schema/manager.go

```diff
@@ -13,10 +13,12 @@ package schema
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"sync"
 
 	"github.com/sirupsen/logrus"
+	"github.com/weaviate/weaviate/cluster/store"
```

Comment:
thought(non-blocking): I'm surprised this doesn't raise any import cycle. Do we want to keep it ?

Reply:
removed it with some refactor